src/map/utils/charutils.cpp

```diff
@@ -6035,7 +6035,7 @@ namespace charutils
         if (strcmp(type, "unity_accolades") == 0 && amount > 0)
         {
             float       evalPoints   = static_cast<float>(amount) / 1000;
-            const char* rankingQuery = "UPDATE unity_system SET points_current = points_current+%f WHERE leader=%d;";
+            const char* rankingQuery = "UPDATE unity_system SET points_current = points_current+(%f) WHERE leader=%d;";
```

Comment:
From your error, I'm going to assume that your code does not match LSB.

```
        const char* Query = "UPDATE char_points SET %s = GREATEST(LEAST(%s+(%d), %d), 0) WHERE charid = %u;";

        sql->Query(Query, type, type, amount, max, PChar->id);

        if (strcmp(type, "unity_accolades") == 0 && amount > 0)
        {
```
The amount value in your logs is <0, and the condition provided requires amount to be >0.  Please confirm that you have checked your local codebase.

Reply:
while our file is a bit out of date (Nov. 18), the functions that handle the accolades are the same.

![image](https://user-images.githubusercontent.com/65316697/206602342-82b0d2b6-8be6-4f65-8355-c8464049e555.png)

Reply:
The only thing that comes to mind at the moment is if you had an _insanely_ high modifier for xp gain, and somehow over 2 billion points were allocated to a single unity leader